packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityBlockEditor } from '@wordpress/core-data';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+export default function TemplatePartInnerBlocks() {
+	const [ blocks, onInput, onChange ] = useEntityBlockEditor(
+		'postType',
+		'wp_template_part',
+		{
+			initialEdits: { status: 'publish' },
+		}
+	);
+	return (
+		<InnerBlocks
+			__experimentalBlocks={ blocks }
```

Comment:
so this is basically the "value" prop right?

Reply:
Yes, but it follows the `BlockEditorProvider` in the naming style.